### ISSUE TITLE
Creation of Italian website.

### ISSUE THREAD

**Issue by Staff91** [rchain/bounties#569]
Hi everyone. I would like to discuss about creating an Italian RChain website. I think it would benefit the Italian community. @Jake-Gillberg @dckc @ian-bloom @lapin7  I would like to hear your opinion about it.

I have purchased an [Italian domain](https://drive.google.com/open?id=14x7NpJu1LdsRPJXWs_EQc7SJ7AyLAVBm) rchaincoop.it and I've secured [hosting](https://drive.google.com/open?id=1O9sgklHAs1B_Rm0TO3lBVUp6cVvyJfE_)  for a year, same as the domain name.
I've created a [facebook page](https://www.facebook.com/rchaincoop.it/) and a [twitter account](https://twitter.com/rchain_it) .
I would like your permission to go through with creating the site and managing the social media. As for the budget, I will set it to zero and, let you set it  to whatever you think is right, based on what you think my work is worth.

**Comment by Barkov-F** [comment]
I have registered a rchain.world domain, hoped that it would be a nice
place to gather all local versions. How about creating a rchain.world/it/ ?
PM on Discord - "ted barkov"

**Comment by Barkov-F** [comment]
Or we can add subdomains it.rchain.world, cn.rchain.world and so on

**Comment by Staff91** [comment]
@Barkov-F I am open to any suggestions, I just want to hear from @ian-bloom @Jake-Gillberg @dckc @lapin7   as well on this matter.

**Comment by Jake-Gillberg** [comment]
I am of the opinion that we should not spin up separate websites for each language until we have a process in place to maintain the infrastructure of a multi-locale website. This should take the from of a localized rchain.coop. I don't support efforts under a different domain.

**Comment by dckc** [comment]
I also prefer one multi-locale website... but...

since we don't have that, I hesitate to discourage work in parallel. Especially when others just go ahead and do it without asking for advice.

I would like to know more about how "it would benefit the Italian community". How big is the community near RChain (developers etc.) that reads Italian but not English? What sort of material would you put on this site? Who would write it? How many new RChain coop members per month would you think is a good goal? Or new developers?

In short:
  1. Tell me some stories about what the world would look like once your idea is up and running, and
  2. propose some test cases by which we could evaluate the work.

**Comment by Staff91** [comment]
@dckc 
1. My personal opinion is that a developer who doesn't know english, is not a developer, full stop.
2. My personal goal is to bring new people in, common folk, businesses (to make use of the contracts on the new RChain platform), to make the features of the RChain platform known to as many people as possible and to make the joining the RChain coop easier for everyone.
My goal isn't to bring in new developers, but to enlarge the target audience of RChain, so that the developers' work can be put to use by more people. 

How will I achieve that? : 
1. With the creation and maintenance of a localized, easy to understand website for RChain, which, by providing blog posts, forums, Q&As, all the documentation and a link to the official RChain website would help  spread knowledge to every potential end-user about the endless possibilities that RChain has to offer.
2. By a professional promotion of RChain on the available social media platforms.


I can't promise miracles, I just want to try doing the best I can to make RChain accessible by everybody. @Jake-Gillberg @ian-bloom @lapin7

**Comment by dckc** [comment]
OK, so the goal isn't to recruit developers; but this bounty program has two goals:

> 1. build a thriving open source developer community
> 2. involve the creative community in telling the RChain story -- [bounties README](https://github.com/rchain/bounties/blob/master/README.md)

and "make the features of the RChain platform known to as many people as possible" is consistent with the 2nd one.

But please give some more thought to SMART objectives; "as many people as possible" is not measurable / attainable.

Also, you might try contributing to the existing RChain blog in parallel. We might learn which draws a bigger audience. Or we might learn that the workflow is just too much of a hassle for somebody like yourself.

**Comment by Staff91** [comment]
I'm already invested in the project, both in time and money. I will make the website irregardless, we will find a way for you to be able to be able to measure the results of my work. If the results are promising, then good for everybody, if not, then it's my loss :) I will open a new issue and we will discuss the specifics over there.
